src/app/url.js

```diff
@@ -2,18 +2,120 @@
 
 const url = {
   // API URL
-  BASE_URL: 'https://the-altcamp.onrender.com',
+  BASE_URL: 'https://api.thealtcamp.com',
 
-  // AUTH URL
+  // AUTH URLS
   LOGIN_URL: '/auth/login',
   REGISTER_URL: '/auth/register',
-  VERIFY_EMAIL_URL: '/auth/start-email-verification',
-  VERIFY_OTP_URL: '/auth/verify-email',
+
+  // EMAIL VERIFICATION URLS
+  START_VERIFY_EMAIL_URL: '/auth/start-email-verification',
+  VERIFY_EMAIL_URL: '/auth/verify-email',
+
+  // FORGOT & RESET PASSWORD URLS
   FORGOT_PASSWORD_URL: '/accounts/forgot-password',
+  VERIFY_FORGOT_PASSWORD_OTP_URL: 'accounts/verify-password-reset-otp',
   RESET_PASSWORD_URL: '/accounts/reset-password',
 
-  // ACCOUNT URL
-  GET_ACCOUNTS_URL: '/accounts',
+  // ACCOUNT URLS
+  ACCOUNTS_URL: '/accounts',
+  UPDATE_PROFILE_PICTURE_URL: '/accounts/profile-picture',
+  UPDATE_BIO_URL: '/accounts/bio',
+  UPDATE_DETAILS_URL: '/accounts',
+  GET_ACCOUNT_BY_ID_URL(url, accountId) {
+    return `${url.ACCOUNTS_URL}/${accountId}`;
+  },
+  GET_SEARCHED_ACCOUNTS_URL(url, searchTerm, isPaginated) {
+    return `${url.ACCOUNTS_URL}?searchTerm=${searchTerm}&isPaginated=${isPaginated}`;
+  },
+  GET_ALL_ACCOUNTS_URL(url, accountType, searchTerm, isPaginated, page, limit) {
+    if (searchTerm && accountType) {
+      return `${url.ACCOUNTS_URL}?category=${accountType}&searchTerm=${searchTerm}&isPaginated=${isPaginated}&page=${page}&limit=${limit}`;
+    } else if (!accountType && searchTerm) {
+      return `${url.ACCOUNTS_URL}?searchTerm=${searchTerm}&isPaginated=${isPaginated}&page=${page}&limit=${limit}`;
+    } else if (accountType && !searchTerm) {
+      return `${url.ACCOUNTS_URL}?category=${accountType}&isPaginated=${isPaginated}&page=${page}&limit=${limit}`;
+    } else {
+      return `${url.ACCOUNTS_URL}?isPaginated=${isPaginated}&page=${page}&limit=${limit}`;
+    }
+  },
```

Comment:
What do you think about using ternary operators to build the URL? 

```js
GET_ALL_ACCOUNTS_URL(url, accountType, searchTerm, isPaginated, page,
  limit) {
  const categoryParam = accountType ? `category=${accountType}&` :
    '';
  const searchTermParam = searchTerm ? `searchTerm=${searchTerm}` :
    '';
  const paginatedParam = `&isPaginated=${isPaginated === true}`;
  const pageParam = (isPaginated === true && page) ?
    `&page=${page}` : '';
  const limitParam = (isPaginated === true && limit) ?
    `&limit=${limit}` : '';

  return `${url.ACCOUNTS_URL}?${categoryParam}${searchTermParam}${paginatedParam}${pageParam}${limitParam}`;
}
```

Reply:
I actually thought about it but decided to go for readability.

Reply:
What I did there previously was ternary innit??